src/Function.cpp

```diff
@@ -574,6 +635,12 @@ Function::OutputForwardDecl(std::ostream &out)
 		int power = rnd_upto(16);
 		out << " __attribute__((aligned(" << (1 << power) << ")))";
 	}
+	vector<FunctionAttribute*>::iterator itr;
+	for(itr = attributes.begin(); itr < attributes.end(); itr++)
+		(*itr)->OutputAttributes(out);
+	if(CGOptions::func_attr_flag())
```

Comment:
I don't like this. You should output `))` when you emitted an attribute.

Reply:
> I don't like this. You should output `))` when you emitted an attribute.

We dont know, how many times the flow will pass `rnd_flipcoin()`. Hence, it is difficult to predict last attribute. That is why I put `))` like this.